### PR TITLE
Tsuru deploy

### PR DESCRIPTION
**Pull request by doctorpod**

### CODE REVIEW

File: lib/mediators.rb [hunk 1]
```diff
@@ -6,4 +7,13 @@
 require_relative 'mediators/entities/mediator'
 require_relative 'mediators/api'
 
-OTR::ActiveRecord.configure_from_file! File.expand_path("../../config/database.yml", __FILE__)
+LOGGER = Logger.new(STDOUT)
+LOGGER.level = Logger::DEBUG
+
+if ENV['DATABASE_URL']
```
Comment (doctorpod) [inline lib/mediators.rb]: Yes - there's a command you run from your local machine that sets env vars --- documentation is at https://tsuru-client.readthedocs.io/en/latest/reference.html#environment-variables

File: requirements.apt [hunk 2]
```diff
@@ -0,0 +1 @@
+libpq-dev
```
Comment (doctorpod) [inline requirements.apt]: I would have added a comment here to explain this file but comments are not allowed in here. Suffice to say that Tsuru uses this file to install any packages that the app requires, one on each line.

**Comment by lores** [comment]
Env variables are the tool of the devil, but aside from that, is there a
way to have Tsuru set them automatically from the repo config? I'm not
clear on how they are set on the remote machine.




*Nicolas Jessus*Sr Technical Architect, MoJ Digital
Mobile: 079186 17498

**Comment by doctorpod** [comment]
Sensitive env vars can be added manually via a local command for each "app" - as I understand it this only has to be done once for each app. Non-sensitive env vars I think can be set automatically by having something like a .env file and calling that with a Tsuru pre-hook (yes Tsuru has hooks).

**Pull request merged**